library/weaviate.py

```diff
@@ -243,16 +243,20 @@ def get_email_metadata_by_id(self, email_id: str) -> Email:
     def get_thread_email_messages_by_id(self, thread_id: str) -> list[EmailTextWithFrom]:
         results = self.collection(WeaviateSchemas.EMAIL_TEXT).query.fetch_objects(
             filters=Filter.by_property("thread_id").equal(thread_id),
-            sort=Sort.by_property(name="date", ascending = True),
+            # sort=Sort.by_property(name="date", ascending = True),
```

Comment:
I've redone this segment entirely, it's much simpler now. See my PR. We can merge it but might be easier if reverted.

Reply:
I've removed it.